core/src/main/java/io/aiven/klaw/repository/AclRequestsRepo.java

```diff
@@ -16,19 +16,24 @@
   List<AclRequests> findAllByTenantId(int tenantId);
 
   @Query(
-      value =
-          "select count(*) from kwaclrequests where env = :envId and tenantid = :tenantId and topicstatus='created'",
+      value = "select count(*) from kwaclrequests where env = :envId and tenantid = :tenantId",
       nativeQuery = true)
   List<Object[]> findAllAclRequestsCountForEnv(
       @Param("envId") String envId, @Param("tenantId") Integer tenantId);
 
   @Query(
-      value =
```

Comment:
yeah because we are getting rid of topicstatus='created' it means a user can never be deleted if they ever create any request i think?

Reply:
yes, removed the 'created', as we will loose some critical user/team/env data if we delete them